tests/unit/amo/components/TestAddonCompatibilityError.js

```diff
@@ -205,6 +206,20 @@ describe(__filename, () => {
     ).toContain('This add-on is not available on your platform.');
   });
 
+  it('renders a notice if add-on is non-restartless', () => {
+    _dispatchClientMetadata();
+    const root = render({ reason: INCOMPATIBLE_NON_RESTARTLESS_ADDON });
+
+    expect(
+      root
+        .find('.AddonCompatibilityError')
+        .childAt(0)
+        .html(),
+    ).toContain(
+      'Your version of Firefox does not support non-restartless add-ons.',
```

Comment:
What do you mean by "legacy"? I would like to avoid any use of the term legacy because I think it is vague and hard to understand. I'd rather see mention of specific features or add-on types that are unsupported.

Reply:
Ah, I just caught up on the thread. My answer is no, I don't think we refer to anything as "legacy" in this case.

Reply:
oki doki